src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -40,4 +42,36 @@ return function (context, callback) {{
     delete context.inputs;
 
     f.apply(null, inputs);
-}};
\ No newline at end of file
+}};
+
+function getEntryPoint(f, context) {{
+    if (util.isObject(f)) {{
```

Comment:
Once C# supports the entryPoint property, the logic for both languages should be identical.